dask/dataframe/core.py

```diff
@@ -2745,6 +2745,41 @@ def _repr_html_(self):
         return self._HTML_FMT.format(data=data, name=key_split(self._name),
                                      task=len(self.dask))
 
+    def _select_columns_or_index(self, columns_or_index):
+        """
+        :param columns_or_index: Column name or list of column names
+        and/or the name of the DataFrame's index
+        :return: Dask DataFrame with columns corresponding to each column or
+        index level in columns_or_index.  If included, the column corresponding 
+        to the index level is named _index
```

Comment:
Dask tends to use numpydoc style docstrings.  http://dask.pydata.org/en/latest/develop.html#docstrings